### DISCUSSION TITLE
Tooling/Libraries: Investigate Ray

### DISCUSSION THREAD

**Discussion by mguidon** [ITISFoundation/osparc-simcore#5581] [Ideas]
Every two years we need to change the computational backend.

**Comment by sanderegg** [comment]
the [docs](https://docs.ray.io/en/latest/)
the [ray clusters part](https://docs.ray.io/en/latest/cluster/getting-started.html) which essentially wraps AWS, GCP and others. This would help a lot in using different platforms without changing the code. Also autoscaling is a given part of the library.
